catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/forms/model/pojo/CommonTemplate.java

```diff
@@ -73,6 +76,8 @@ public CommonTemplate(Metacard metacard, Map<String, List<Serializable>> securit
     this.accessIndividuals =
         securityAttributes.getOrDefault(Security.ACCESS_INDIVIDUALS, new ArrayList<>());
     this.accessGroups = securityAttributes.getOrDefault(Security.ACCESS_GROUPS, new ArrayList<>());
+      this.accessAdministrators =
+              securityAttributes.getOrDefault(Security.ACCESS_ADMINISTRATORS, new ArrayList<>());
```

Comment:
✏️ should we change new ArrayList<>() in these guys to be Collections.emptyList?